styles/style-guide.scss

```diff
@@ -5,6 +5,7 @@
 @import 'base';
 @import 'type';
 @import 'iconography';
+@import 'identity';
```

Comment:
I am not sure but I think in ppm we import this file fully? maybe? Not sure if PPM uses .daptiv-logo classes and any impact on it..